pkg/bootstrap/os/tasks.go

```diff
@@ -454,3 +454,32 @@ func (u *UmountISO) Execute(runtime connector.Runtime) error {
 	}
 	return nil
 }
+
+type NodeConfigureNtpServer struct {
+	common.KubeAction
+}
+
+func (n *NodeConfigureNtpServer) Execute(runtime connector.Runtime) error {
+
+	for _, server := range n.KubeConf.Cluster.System.NtpServers {
```

Comment:
I think if no set time server, we can selecte one  node in the cluster as the default time server, and other nodes can synchronize time from this node. What about you ?

Reply:
ok i think  we can  select master node as ntp servers to avoid single point of failure.

Reply:
@pixiake  now support use k8s node to configure the ntp server, please review again.

Reply:
What happened when the length of ntpServer is 0?

Reply:
> What happened when the length of NtpServers is 0? Do nothing, just like skip this task? Maybe you can add a `prepare` to this task. That will make the log print more graceful, because they will show `[node1]:skipped` when the condition is false.

got, i had fix this with add `NodeConfigureNtpCheck` to check the config, please review again @24sama

Reply:
Ok, Thanks again. @tanguofu